python3/packages/arranger_cdk8s/arranger_cdk8s/arranger_services/_basic_gateway.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, scope: Construct, id: str, *, config, kwargs=None):
         self.config = config
         self.kwargs = kwargs
         self.environment = self.kwargs.get("environment")
-        self.globals = Cdk8sGlobals(environment=self.environment, config=config)
+        self.globals = Cdk8sGlobals(sub_environment=self.environment, config=config)
```

Comment:
typo?

Reply:
this will be refactored in the nearest future